src/js/actions/staff.js

```diff
@@ -21,6 +24,37 @@ const keyOffDate = (availableSlots) => {
     return formattedSlots;
 };
 
+export const fetchSlots = startDate =>
+    (dispatch, getState) => {
+        const month = startDate.month();
+        const anyResourceId = Object.keys(getState().staff.allAvailableSlots)[0];
```

Comment:
Do we really want to get the resource id by expecting it at first position every time?  Rather than by key name?